mypy/checkmember.py

```diff
@@ -33,28 +33,17 @@ def analyze_member_access(name: str,
                           not_ready_callback: Callable[[str, Context], None],
                           msg: MessageBuilder,
                           override_info: TypeInfo = None,
-                          actual_self: Type = None,
+                          original_type: Type = None,
                           chk: 'mypy.checker.TypeChecker' = None) -> Type:
-    """Analyse attribute access.
+    """Return the type of attribute `name` of typ.
 
-    This is a general operation that supports various different variations:
-
-      1. lvalue or non-lvalue access (i.e. setter or getter access)
```

Comment:
Please retain `This is general ... ` and the following numbered list as they are still relevant.